### ISSUE TITLE
Cannot make ethernet to work

### ISSUE THREAD

**Issue by bfgelectronics** [radxa/radxa-cm-projects#21]
Ive got my Radxa CM3 and i flashed on a sd card the debian and ubuntu servers images for the E23 board.
When i power up the E23 carrier board with the CM3 attached , the led from the CM3 starts blinking up , but when i connect the ethernet cable on one of the ethernet ports, the amber let lights up and blinks, but it isnt detected by my router. If i insert the same cable on the other ethernet port , the green led lights up and same beheaviour.
I tryed connecting directly a ethernet port to my laptop and it seems that it the laptop tryes to connect , but after a few seconds, it disconects and the process repeats.

TLDR: I tryed using the images for the e23 boards and ethernet is not working

**Comment by tomek-szczesny** [comment]
The fact your laptop tries to connect but it fails might either mean misconfiguration (or lack of sane configuration), or something wrong with a cable.
I'd assume the drivers are fine, and since it's not a prototype carrier board anymore (I think?), I'd rule out design errors.

**Comment by bfgelectronics** [comment]
Thanks for the answer @tomek-szczesny the cable is fine. Also since the e23 board has no video out, i couldnt make any configuration, i just downloaded the 2 mentioned images, burned them to a sd card, and tryed them.

**Comment by tomek-szczesny** [comment]
I can't help you any more than that, I'm still waiting for my unit, and the carrier board I'm designing isn't ready yet :)

Perhaps it'd be better idea to ask for directions on Radxa forum instead.

Good luck with your project! :)
